app/selenium_ui/conftest.py

```diff
@@ -48,17 +49,6 @@ def __get_current_results_dir():
         pass
 
 
-def application_url(yml_file_path: Path):
-    with yml_file_path.open(mode='r') as fs:
-        jira_yaml = yaml.load(fs, Loader=yaml.FullLoader)
```

Comment:
you deleted yml file do not forget delete unused imports (`ctrl+alt+o` in pycharm)

Reply:
Thx for the shortcut, very useful!